code/src/pages/products/ProductsPage.tsx

```diff
@@ -31,6 +30,7 @@ const ProductsPage = () => {
     isLoading,
     isError
   } = useGetUserProductsQuery({
+    tags: categoryType ? `category:${categoryType}` : "",
```

Comment:
Please attach the wideo to show how it works now?

Reply:
Pretty much, the endpoint accepts a new property called tags. The tag is a string in the form of "category:<category>". If we pass this string we get all products that have this tag. If we pass an empty string, we get list of all products. It is a bit unintuitive, because we have to pass a string in this format and not just the name of the category, but that is how backend team have implemented it.